read.txt

```diff
@@ -0,0 +1 @@
+#thueibio
```

Comment:
This is the worst txt file I have ever seen in my life

Reply:
This is a safe space Samuel. No need for this kind of negativity.
![We_Don't_Do_That_Here](https://user-images.githubusercontent.com/14300728/74088502-cc1b2880-4a5c-11ea-8adc-c04952ec4875.jpg)